doc/book/src/development/release.md

```diff
@@ -0,0 +1,29 @@
+# Making a release
```

Comment:
I'd suggest taking one sentence per line. I find it way easier to read. Does it work for you?

Reply:
In my recent writing, I've more taken to setting a text-width of 100 columns (in vim/nvim, `set tw=100`), which is what I've used here.  I've found it easier to both read & write this way.

@dperaza4dustbit do you have any opinions on a documentation style convention?  I'm not completely set on what I do; at the end of the day, it's getting rendered anyway.